collector/src/compile/benchmark/target.rs

```diff
@@ -5,6 +5,7 @@ use std::{fmt, str::FromStr};
 /// https://doc.rust-lang.org/nightly/rustc/platform-support.html
 ///
 /// Presently we only support x86_64
+/// FIXME: we actually support Windows and aarch64, but that isn't captured here.
```

Comment:
This Target enum is not exactly "what does rustc-perf support benchmarks on", but "what targets we support/run in production".